README.md

```diff
@@ -0,0 +1,56 @@
+# YOUTUBE HOMEPAGE
+
+> This is a project aimed at implementing lessons learned on HTML5 AND CSS3. The project provides a simple layout of the Youtube homepage and affords me the opportunity to learn and implement what i learnt.
+
+![screenshot](src/assets/images/screenshot.png)
+
+## Built With
+
+- HTML5
+- CSS3
+
+## Live Demo
+
+[YOUTUBE HOMEPAGE](https://fodieshub.github.io/youtubehomepage/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- On the project GitHub page, navigate to the main page of the repository [this page](https://github.com/fodieshub/youtubehomepage).
+- Under the repository name, locate and click on a green button named `Code`.
+- Copy the project URL as displayed.
+- If you're running Windows Operating System, open your command prompt. On Linux, Open your terminal.
+- Change the current working directory to the location where you want the cloned directory to be made. Leave as it is if the current location is where you want the project to be.
+- Type `git clone`, and then paste the URL you copied in Step 3.<br>
+  `$ git clone https://github.com/fodieshub/youtubehomepage.git` <em>Press Enter key</em><br>
+- Press Enter. Your local copy will be created.
+
+Please Note that you must have github installed on your PC, this can be done [here](https://gist.github.com/derhuerst/1b15ff4652a867391f03).
+
+### Prerequisites
+
+Web browser (Chrome/Firefox)
+
+## Author
+
+👤 *Audu Kehinde*
+
+- Github: [@fodieshub](https://github.com/fodieshub)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Taofeek Olalere](https://github.com/teekaytech)
+- [The Odin Project (Project Spec)](https://www.theodinproject.com/courses/javascript/lessons/todo-list)
```

Comment:
This is not the correct link to the project you built. Please review and make corrections as appropriate.

Reply:
I was trying to acknowledge the odin project